From 6ffa575a9af50da6da4e98f16dda2f5e28782676 Mon Sep 17 00:00:00 2001
From: Chris Gorman <chrisjohgorman@gmail.com>
Date: Thu, 27 Jun 2024 05:26:41 -0400
Subject: [PATCH 2/2] Removing the use of os.path.join when getting the path to
 grc_qt.conf

Using os.path.join breaks the path under windows requiring a settting
of GRC_QT_PREFS_PATH to point to the file or the conf file doesn't
work.
---
 grc/gui_qt/Config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grc/gui_qt/Config.py b/grc/gui_qt/Config.py
index ec5fffccc..664092b2b 100644
--- a/grc/gui_qt/Config.py
+++ b/grc/gui_qt/Config.py
@@ -12,7 +12,7 @@ class Config(CoreConfig):
     name = 'GNU Radio Companion'
 
     gui_prefs_file = os.environ.get(
-        'GRC_QT_PREFS_PATH', os.path.join(get_config_file_path(), 'grc_qt.conf'))
+        'GRC_QT_PREFS_PATH', get_config_file_path('grc_qt.conf'))
 
     def __init__(self, install_prefix, *args, **kwargs):
         CoreConfig.__init__(self, *args, **kwargs)
-- 
2.45.2

